### PR TITLE
[VBLOCKS-1580] fix: detox with auth0

### PR DESCRIPTION
**Pull request by kpchoy**
[VBLOCKS-1580](https://issues.corp.twilio.com/browse/VBLOCKS-1580)

Update build.gradle with subproject auth0 

Reference: https://github.com/auth0/react-native-auth0/issues/387#issuecomment-918483390

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: app/android/build.gradle [hunk 1]
```diff
@@ -55,3 +55,19 @@ allprojects {
...
+        if (subproject.name == 'react-native-auth0') {
+            def String taskRequests = getGradle().getStartParameter().getTaskRequests().toString()
+            if (taskRequests.contains("assembleAndroidTest")) {
+                android {
+                    defaultConfig {
+                        manifestPlaceholders = [auth0Domain: "E2E_TEST", auth0Scheme: "com.E2E.TEST"]
+                    }
+                }
+            }
+        }
```
Comment (charliesantos) [inline app/android/build.gradle:71]: Will any of these values change if another user sets up their own app?

Comment (charliesantos) [reply app/android/build.gradle:71]: Can you add a comment right above this code block explaining why you had to do this? And also add the link to https://github.com/auth0/react-native-auth0/issues/387#issuecomment-918483390

Customers might wonder what this is for.

Comment (kpchoy) [reply app/android/build.gradle:71]: @charliesantos yes I did, already on line 59? Or did you want to change the text?

Comment (charliesantos) [reply app/android/build.gradle:71]: Sorry missed that. +1

**Review by charliesantos — APPROVED** [review]

**Pull request merged**